### DISCUSSION TITLE
Interpret ellipsis (…) as 3 periods (...)

### DISCUSSION THREAD

**Discussion by SethFalco** [monkeytypegame/monkeytype#2208] [Ideas] [answered]
Some quotes contain an ellipsis, but Monkeytype translates them to `...` (3 periods). This make sense for most users, but for some like me, it's a bit annoying since I actually type an ellipsis.

To get around this, when a user types an ellipsis character (`…`), could it be interpreted as 3 periods? (Preferably just when there are 3 periods following the user's cursor, but otherwise across the entire typing test is fine.)

I do an ellipsis using the compose key on GNOME. (`COMPOSE` `.` `.`)

I'd be happy to try and make the PR, but just wanted to check on the opinion on this first.

More information on the compose key:
https://en.wikipedia.org/wiki/Compose_key#Common_compose_combinations

**Comment by toastli** [comment]
Just to add to this, people on ios devices, the three periods automatically get corrected to the unicode character (`…`).  This makes it very annoying to type (`...`)

**Comment by Miodec** [comment, marked as answer]
Created #2218
